src/main/java/com/amazon/aocagent/testamis/LinuxAMI.java

```diff
@@ -18,8 +25,8 @@ public String getAMIId() {
   }
 
   @Override
-  public OSType getOSType() {
-    return OSType.LINUX;
+  public boolean isUseSSM() {
```

Comment:
i think we don't need this method? because we configure the installer as ssm installer already. in the future if we do linux with ssm as well, this flag will be hard to use. because we need to decide the install mechanism in testcase level.